specifyweb/frontend/js_src/lib/components/FormEditor/viewSpec.ts

```diff
@@ -88,12 +88,32 @@ const rowsSpec = (table: SpecifyTable | undefined) =>
                 } as const,
                 table
               ),
-              // Make sure command is on a correct table
               syncers.change(
                 'definition',
                 (cell) => {
+                  if (
+                    cell.definition.type === 'SubView' &&
+                    cell.definition.sortField !== undefined
+                  ) {
+                    // Finish parsing sortField based on the related table name
+                    const parsed = toLargeSortConfig(cell.definition.sortField);
+                    const fieldNames = syncers
+                      .field(cell.definition.name?.at(-1)?.table.name)
```

Comment:
user might have provided us with multiple field names (separated by a dot), which after parsing is an array, so accessing the last array item here (`.at(-1)`)